packages/@lwc/errors/src/compiler/error-info/template-transform.ts

```diff
@@ -633,4 +633,17 @@ export const ParserDiagnostics = {
         level: DiagnosticLevel.Warning,
         url: '',
     },
+    INVALID_OPTS_LWC_SPREAD: {
+        code: 1154,
+        message:
+            'Invalid lwc:spread usage. The `lwc:spread` Directive must be enabled in order to use this feature.',
+        level: DiagnosticLevel.Error,
+        url: '',
+    },
+    INVALID_LWC_SPREAD_LITERAL_PROP: {
+        code: 1155,
+        message:
+            'Invalid lwc:spread usage on element "{0}". The directive binding must be an expression.',
+        level: DiagnosticLevel.Error,
+    },
```

Comment:
What happens if `lwc:spread` is used on a `<template>`? E.g. a `<template for:each>`?